src/app/src/main/kotlin/org/vaccineimpact/api/app/controllers/GroupResponsibilityController.kt

```diff
@@ -56,12 +59,17 @@ class GroupResponsibilityController(
         return data
     }
 
-    fun getTemplate()
+    fun getTemplate(): StreamSerializable<Any?>
     {
         val path = ResponsibilityPath(context)
         modellingGroupRepo.getModellingGroup(path.groupId)
         val expectations = expectationsLogic.getExpectationsForResponsibility(path.groupId,
                 path.touchstoneVersionId, path.scenarioId)
+
+        val headers = arrayOf("disease", "year", "age", "country", "countryName", "cohortSize") +
```

Comment:
Wasn't sure whether to pull these from `BurdenEstimate` by reflection, or just hard-code here :woman_shrugging:

Reply:
How about you make `EmptyDataTable` a bit more like `DataTable` - give it a type to derive headers from, and reuse the logic for that? You can still then just generate a number of blank rows off the back of that.